.gitignore

```diff
@@ -1,3 +1,5 @@
+# Ignores any `development` Folder
+**/**/development/
```

Comment:
I assume this is something you used internally. Can you please remove it?

Reply:
Yes sorry, that was only for development purposes, so that i don't accidentally push something, i don't want to. Removed it from the history.